### PR TITLE
Fix CI on feature branch

### PR DESCRIPTION
**Pull request by newswangerd**

### CODE REVIEW

File: .github/template_gitref [hunk 1]
```diff
@@ -1 +1 @@
-2021.08.26-106-g556224f
+2021.08.26-129-gf780fda-dirty
```
Comment (awcrosby) [inline .github/template_gitref:1]: May need to rebase against master, we are already at `f780fda`

https://github.com/ansible/galaxy_ng/blob/8bbb7a58aa9f64ad3afaf6c4ba9cb46611d3ff5b/.github/template_gitref#L1

**Review by awcrosby — CHANGES_REQUESTED** [review]
May need to rebase

File: .github/template_gitref (hunk 1 above)
Comment (awcrosby) [reply .github/template_gitref:1]: oh nevermind, you are updating this feature branch... why does this say `-dirty`? ... i have not seen that

Comment (newswangerd) [reply .github/template_gitref]: I think i screwed up my plugin template checkout

**Comment by newswangerd** [comment]
>

**Comment by newswangerd** [comment]
> May need to rebase

I'd have to rebase the whole feature branch against master

**Pull request merged**